app/scripts/background.js

```diff
@@ -1349,6 +1365,7 @@ async function initBackground() {
     persistenceManager.cleanUpMostRecentRetrievedState();
   } catch (error) {
     log.error(error);
+    rejectInitialization(error);
```

Comment:
bubbles ALL initialization errors, not just the ones in `await initialize()` like we were doing before. 

I don't think will actually change anything, since I don't think the other functions `initBackground` calls will throw, but I don't think it is better suited here.